pytorch_lightning/loops/dataloader/evaluation_loop.py

```diff
@@ -110,7 +120,8 @@ def on_run_start(self, *args: Any, **kwargs: Any) -> None:
         hooks."""
         void(*args, **kwargs)
 
-        self._data_fetcher = DataFetcher()
+        data_fetcher_cls = _select_data_fetcher_type(self.trainer)
+        self._data_fetcher = data_fetcher_cls()
```

Comment:
@carmocca - n00b question: why return the type vs the instance in #11621 
 ?

here i'm simply copy/pasting that style but was wondering if you had a specific reason its done this way: 

https://github.com/PyTorchLightning/pytorch-lightning/blob/f79d75d4b5e50eb3e37073606eba5ca10d48c99a/pytorch_lightning/loops/fit_loop.py#L346-L358

Reply:
In case one wants to modify the number of prefetch batches passed.